python/cuopt/cuopt/tests/linear_programming/test_python_API.py

```
@@ -968,74 +737,4 @@ def test_quadratic_matrix_2():
-    settings.set_parameter(CUOPT_MIP_BATCH_PDLP_STRONG_BRANCHING, 0)
-
-    problem.solve(settings)
-    assert problem.Status.name == "Optimal"
-    assert problem.ObjValue == pytest.approx(-126, abs=1e-3)
-
-    settings.set_parameter(CUOPT_MIP_BATCH_PDLP_STRONG_BRANCHING, 1)
-
-    problem.solve(settings)
-    assert problem.Status.name == "Optimal"
-    assert problem.ObjValue == pytest.approx(-126, abs=1e-3)
+    assert problem.ObjValue == pytest.approx(3.715847, abs=1e-3)
```

Ruff (0.15.10)

[warning] No newline at end of file

Add trailing newline

(W292)